pkg/controller/masteruserrecord/sync.go

```diff
@@ -220,6 +222,27 @@ func (s *Synchronizer) alignReadiness() bool {
 			return false
 		}
 	}
+
+	s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, toBeProvisioned())
+	if condition.IsNotTrue(s.record.Status.Conditions, toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated) {
+		notification := &toolchainv1alpha1.Notification{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      s.record.Name + "-provisioned",
+				Namespace: s.record.Namespace,
+			},
+			Spec: toolchainv1alpha1.NotificationSpec{
+				UserID:   s.record.Spec.UserID,
+				Template: "userprovisioned",
+			},
+		}
+
+		if err := s.hostClient.Create(context.TODO(), notification); err != nil {
+			s.logger.Error(err, "failed to create user provisioned notification")
```

Comment:
logging the error is not enough. I think instead of logging we should return an error here (wrap the original error) and wrap it with the status in the controller.
So it will be logged by the controller, error status will be added, the reconcile will be re-queued.

Reply:
sounds good, it looks like the controller already wraps the error, is this what you were looking for: 00a78eca82abc149ee7203742b9fe16ac7016476?

Reply:
Yes. But we don't need to log this error here. It will be logged by the controller.

Reply:
fixed as part of: 43b2953aa3f7922169954600dd7d4cabfbe0a2ac